go/client/cmd_simplefs.go

```diff
@@ -70,6 +73,13 @@ func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
 
 	path = filepath.ToSlash(filepath.Clean(path))
 
+	// Certain users seem to want to use SimpleFS on their
+	// mounted KBFS. This is for those who want to do so
+	// from "/keybase/..."
+	if strings.HasPrefix(path, mountDir) {
+		return keybase1.NewPathWithKbfs(path[len(mountDir):])
```

Comment:
There has been a lot of discussion about identifying when the local dir is actually a KBFS mount, and there is not a really solid way to do it on all the platforms. For now, a path starting with "/keybase" is KBFS.